test_infra/src/lib.rs

```diff
@@ -249,7 +249,8 @@ impl DiskConfig for UbuntuDiskConfig {
             .unwrap()
             .join("test_data")
             .join("cloud-init")
-            .join("ubuntu");
+            .join("ubuntu")
+            .join("ci");
```

Comment:
I think it is better to have this change in a separate commit with moving the original cloud-init data into a new folder.

Reply:
Isn't this commit giving nice and complete information about the changes.

```
---
 README.md                                           |  4 +++-
 scripts/create-cloud-init.sh                        |  6 +++---
 test_data/cloud-init/ubuntu/{ => ci}/meta-data      |  0
 test_data/cloud-init/ubuntu/{ => ci}/network-config |  0
 test_data/cloud-init/ubuntu/{ => ci}/user-data      |  0
 test_data/cloud-init/ubuntu/{ => local}/meta-data   |  0
 test_data/cloud-init/ubuntu/local/network-config    |  5 +++++
 test_data/cloud-init/ubuntu/local/user-data         | 10 ++++++++++
 test_infra/src/lib.rs                               |  3 ++-
 9 files changed, 23 insertions(+), 5 deletions(-)
 copy test_data/cloud-init/ubuntu/{ => ci}/meta-data (100%)
 rename test_data/cloud-init/ubuntu/{ => ci}/network-config (100%)
 rename test_data/cloud-init/ubuntu/{ => ci}/user-data (100%)
 rename test_data/cloud-init/ubuntu/{ => local}/meta-data (100%)
 create mode 100644 test_data/cloud-init/ubuntu/local/network-config
 create mode 100644 test_data/cloud-init/ubuntu/local/user-data
```

Reply:
It is a good practice to keep each commit focused while being self-contained. I can see this commit does two separate things:

1. Move the current cloud-init data to a new folder so that it will be exclusively used for CI;
2. Add new cloud-init data specifically for manual testing.

Reply:
done, split into two commits.